src/test/java/com/snack/news/controller/SlackBotControllerTest.java

```diff
@@ -0,0 +1,83 @@
+package com.snack.news.controller;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.snack.news.exception.SlackAuthorizationException;
+import com.snack.news.exception.advice.ControllerExceptionHandler;
+import com.snack.news.service.SlackBotService;
+import com.snack.news.util.SlackAuthHttpClient;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+import org.springframework.web.filter.CharacterEncodingFilter;
+
+import static com.snack.news.controller.ApiUrl.Domain.SLACK_AUTHORIZE;
+import static java.nio.charset.StandardCharsets.UTF_8;
+import static org.hamcrest.Matchers.containsString;
+import static org.mockito.ArgumentMatchers.*;
+import static org.mockito.Mockito.*;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@ExtendWith(MockitoExtension.class)
+public class SlackBotControllerTest {
+	@InjectMocks
+	private SlackBotController slackBotController;
+
+	@Mock
+	private SlackBotService slackBotService;
+
+	@Mock
+	private SlackAuthHttpClient.Response slackAuthHttpClientResponse;
+
+	private MockMvc mockMvc;
+
+	@BeforeEach
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/snack-news/Snack-BE/pullRequest?prid=5471424)